sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 - Renamed the model `Appearance` to `TextAppearance`.
 - Renamed the model `Style` to `TextStyle`.
 - Renamed the extensible enum `TextStyle` to `TextStyleName`.
+- Changed model type for `Locale` from `string` to `FormRecognizerLocale` for `RecognizeBusinessCardsOptions`, `RecognizeInvoicesOptions`, and `RecognizeReceiptsOptions`.
+- Changed model type for `Language` from `string` to `FormRecognizerLanguage` for `RecognizeContentOptions`.
```

Comment:
They're definitely not wrong, but it was a bit hard to parse and understand these sentences, specially the first one. Maybe changing the prepositions to:
```
Changed model type of `Locale` from `string` to `FormRecognizerLocale` in `Recognize...`, `Recognize...`, and ...
```

Reply:
Thank you!